ui/src/components/ClubsCarousel/index.tsx

```diff
@@ -1,37 +1,46 @@
 import styles from './styles.module.scss';
 import Carousel from 'react-multi-carousel';
 import 'react-multi-carousel/lib/styles.css';
-import {dummyClubLogo} from '../../assets';
-
-const data = [
-  {
-    name: 'TPC',
-    category: 'Technical',
-    logoUrl: dummyClubLogo,
-  },
-  {
-    name: 'Jazbaat',
-    category: 'Cultural',
-    logoUrl: dummyClubLogo,
-  },
-  {
-    name: 'Basketball',
-    category: 'Sports',
-    logoUrl: dummyClubLogo,
-  },
-  {
-    name: 'Saaz',
-    category: 'Cultural',
-    logoUrl: dummyClubLogo,
-  },
-  {
-    name: 'Badminton',
-    category: 'Sports',
-    logoUrl: dummyClubLogo,
-  },
-];
+import {useEffect, useState} from 'react';
+import {useNavigate} from 'react-router-dom';
 
 export const ClubsCarousel: React.FC = () => {
+  interface ClubsCarouselProps {
```

Comment:
These are not Carousel props, so naming them so doesn't make sense. Maybe `ClubsCarouselData` sounds better?